### DISCUSSION TITLE
Ignore remote egress, possible?

### DISCUSSION THREAD

**Discussion by cedric-marcoux** [gravitl/netmaker#2130] [Q&A]
Hello, is this possible to join a network but ignore shared egress gateway network when joining?

Thank you

**Comment by mattkasun** [comment]
You could use an acl to disable communication between the new node and egress gateway

**Reply by cedric-marcoux** [reply]
Hello, thank you.
I have already try but it block entire access to the node who is the egress gateway, i would like to be able to just ignore propagated route from this egress node...
